scripts/build_demand_profiles.py

```diff
@@ -109,15 +110,29 @@ def get_load_paths_gegis(ssp_parentfolder, config):
     ssp = config.get("load_options")["ssp"]
 
     load_paths = []
-    for continent in region_load:
-        load_path = os.path.join(
-            ssp_parentfolder,
-            str(ssp),
-            str(prediction_year),
-            "era5_" + str(weather_year),
-            str(continent) + ".nc",
-        )
-        load_paths.append(load_path)
+    load_dir = os.path.join(
+        ssp_parentfolder,
+        str(ssp),
+        str(prediction_year),
+        "era5_" + str(weather_year),
+    )
+
+    for ext in [".nc", ".csv"]:
+        avail_regions = [
+            cnt
+            for cnt in region_load
+            if os.path.exists(os.path.join(load_dir, cnt + ext))
+        ]
+        if len(avail_regions) == len(region_load):
+            for continent in region_load:
+                load_path = os.path.join(load_dir, continent + ext)
+                load_paths.append(load_path)
+            break
```

Comment:
Could you try this?
I think the two for loops should be inverted (first the continent then the extension).

```suggestion
    for continent in region_load:
        for ext in [".nc", ".csv"]:
            load_path = os.path.join(load_dir, continent + ext)
            if os.path.exists(load_path):
                load_paths.append(load_path)
            break
```

Reply:
Note that this approach may mix csv and nc files though